pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/index.tsx

```diff
@@ -300,7 +345,9 @@ export class NodeGraphs extends React.Component<
 
     // Generate graphs for the current dashboard, wrapping each one in a
     // MetricsDataProvider with a unique key.
-    const graphs = dashboards[dashboard].component(dashboardProps);
+    const graphs = dashboards[dashboard]
+      .component(dashboardProps)
+      .filter(d => (canViewKvGraphs ? true : !d.props.isKvGraph));
```

Comment:
I think the ternary can be replaced with `||`

Reply:
Done.